public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -189,6 +188,8 @@ export class EhrViewComponent implements OnInit, OnDestroy {
     }
   }
   public selectGraph(g) {
+    document.getElementById('chartElement').scrollIntoView(
```

Comment:
This breaks component encapsulation. This would be better:

html:
```
<div #chartElement>
```

ts:
```
@ViewChild('chartElement') chartEl: ElementRef;
```

Reply:
Done.